Change one example subheading and added missing line in map.md for Go (#1371)
Codecademy/docs: content/go/concepts/map/map.md

85.2
```

> ### Store a value accessed by its corresponding key into a varable

Reviewer: ```suggestion
### Access and store a key-value pair in a variable
```

I think the original title makes more sense
